subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -9,27 +9,39 @@
 #include <toolchain.h>
 #include <sys/util.h>
 
+#include <zephyr.h>
+
+#include <bluetooth/bluetooth.h>
+
 #include "util/memq.h"
 #include "pdu.h"
+
+
+#include "ll.h"
 #include "lll.h"
+#include "lll_conn_iso.h"
 #include "isoal.h"
+#include "ull_iso_types.h"
 
 #define LOG_MODULE_NAME bt_ctlr_isoal
 #include "common/log.h"
 #include "hal/debug.h"
 
 /* TODO this must be taken from a Kconfig */
 #define ISOAL_SINKS_MAX   (4)
+#define ISOAL_SOURCES_MAX (4)
```

Comment:
Suggest to create Kconfigs now

Reply:
Come to think of it, we will need a separate Kconfig for this. A device could have one microphone and one speaker, i.e., one source and one sink, but support uplink and downlink in separate CISes. Attempts at guessing how many streams a device needs to support will inevitably go wrong, so this needs to be a separate configuration. For now, leaving it as is is fine by me.